app/errors.js

```diff
@@ -8,3 +8,6 @@ exports.databaseError = message => internalError(message, exports.DATABASE_ERROR
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.BAD_GATEWAY = 'bad_gateway';
```

Comment:
Here It is better to rename as `exports.BAD_GATEWAY_ERROR` in order to specify that variable as an error :)

Reply:
Done!